pkg-example-amf-bp/README.md

```diff
@@ -0,0 +1,21 @@
+# pkg-amf
+
+## Description
+example pkg for amf blueprint
+
+## Usage
```

Comment:
I would just remove the Usage sections, and expand on the description to describe what the package is for. "Used to deploy a free5gc AMF..." etc.

Eventually, in here we would describe if this package accepts any inputs, config injection etc.

Same comment for the SMF

Reply:
agree I would keep the part that is related to how to use the package.

maybe make a reference to the process this package should go through to reach deployment. Not all details but the high level parts